src/middleware/queriesV2.ts

```diff
@@ -319,6 +319,70 @@ export function instructionsQuery(
   };
 }
 
+/**
+ * @hidden
+ *
+ * Get Instructions where an identity is involved
+ */
+export function instructionsByDidQuery(identityId: string): GraphqlQuery {
+  const query = gql`
+    query InstructionsByDidQuery
+     {
+      legs(
+        filter: { or: [{ fromId: { startsWith: "${identityId}" } }, { toId: { startsWith: "${identityId}" } }] }
+        orderBy: [${LegsOrderBy.CreatedAtAsc}, ${LegsOrderBy.InstructionIdAsc}]
+      ) {
+        nodes {
+          instruction {
+            id
+            eventIdx
+            eventId
+            status
+            settlementType
+            venueId
+            endBlock
+            tradeDate
+            valueDate
+            legs {
+              nodes {
+                fromId
+                from {
+                  identityId
+                  number
+                }
+                toId
+                to {
+                  identityId
+                  number
+                }
+                assetId
+                amount
+                addresses
+              }
+            }
+            memo
+            createdBlock {
+              blockId
+              hash
+              datetime
+            }
+            updatedBlock {
+              blockId
+              hash
+              datetime
+            }
+          }
+        }
+      }
+    }
+  `;
+
+  return {
+    query,
+    variables: undefined,
```

Comment:
(nit) Should `identityId` be a GraphQL variable? 

Looks like its getting string interpolated which doesn't seem as nice as the first class variable to me.

That said, if it needs to be like that for some reason or is more consistent its OK to leave